src/interfaces/filing-interfaces/filing-interface.ts

```diff
@@ -52,6 +52,7 @@ export interface IncorporationFilingBodyIF {
     filingId?: number // Optional as this is not required when building a filing - causes an error for new filings
     folioNumber?: string // Optional to the user and only displayed for certain account types
     isFutureEffective: boolean
+    status?: string
```

Comment:
To keep track of whether this a draft or filed.
I mean, it's not impossible that someone enters a draft IA id in the URL, and this app would load it fine probably.

Reply:
Actually I'm going to commit another change to check this.